deploy/kubeeye.yaml

```diff
@@ -387,7 +387,7 @@ spec:
         - --upstream=http://127.0.0.1:8080/
         - --logtostderr=true
         - --v=0
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+        image: gcr.lank8s.cn/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
Is it better to use the  official repo for `kube-rbac-proxy` by default?
`gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0` or `kubesphere/kube-rbac-proxy:v0.8.0`

Reply:
gcr.io is limited access for CN user，we consider replacing it.

Reply:
I think we can replace it with `kubesphere/kube-rbac-proxy:v0.8.0`.